### PR TITLE
feat(composer): use getcomposer.org with checksum

### PR DESCRIPTION
**Pull request by Chumper**

### CODE REVIEW

File: src/usr/local/buildpack/tools/v2/maven.sh [hunk 1]
```diff
@@ -9,6 +9,9 @@ function install_tool () {
   local versioned_tool_path
   versioned_tool_path=$(create_versioned_tool_path)
 
+  local checksum_file
+  local original_checksum
+
```
Comment (viceice) [inline src/usr/local/buildpack/tools/v2/maven.sh:14]: unrelated change?

File: src/usr/local/buildpack/tools/v2/node.sh [hunk 2]
```diff
@@ -41,10 +41,13 @@ function install_tool () {
   local npm # temp npm executable
   local arch=linux-x64
 
-  checksums=$(get_from_url "https://nodejs.org/dist/v${TOOL_VERSION}/SHASUMS256.txt")
+  local checksum_file
+  local original_checksum
+
+  checksum_file=$(get_from_url "https://nodejs.org/dist/v${TOOL_VERSION}/SHASUMS256.txt")
```
Comment (viceice) [inline src/usr/local/buildpack/tools/v2/node.sh:47]: didn't we merged that already?

File: src/usr/local/buildpack/tools/v2/maven.sh (hunk 1 above)
Comment (Chumper) [reply src/usr/local/buildpack/tools/v2/maven.sh:14]: yeah, I unified the changes, so that the naming is consistent between each tool when we fetch the checksums

File: src/usr/local/buildpack/tools/v2/node.sh (hunk 2 above)
Comment (Chumper) [reply src/usr/local/buildpack/tools/v2/node.sh:47]: And I am not up to date, let me rebase...

Comment (Chumper) [reply src/usr/local/buildpack/tools/v2/node.sh:47]: I am... disregard my last comment.
As I said, I changed the naming to be consistent

File: src/usr/local/buildpack/tools/v2/maven.sh (hunk 1 above)
Comment (viceice) [reply src/usr/local/buildpack/tools/v2/maven.sh:14]: i don't see usage of those new variables. please create a refactor PR first.

File: src/usr/local/buildpack/tools/v2/node.sh (hunk 2 above)
Comment (viceice) [reply src/usr/local/buildpack/tools/v2/node.sh:47]: please let's do that in separate refactor PR
